.github/workflows/build.yml

```
@@ -20,7 +20,7 @@ jobs:
       - name: Setup .NET Core SDK
```

```suggestion
      - name: Setup .NET SDK
```

and for other workflows